content/2025/china/_index.zh.html

```diff
@@ -0,0 +1,34 @@
+---
+title: "JakartaOne China 2025"
+seo_title: "JakartaOne China - 25 July 2025 | Jakarta EE Software | Cloud Native"
+headline: "JakartaOne China 2025"
+hide_page_title: true
+hide_sidebar: true
+hide_breadcrumb: true
+container: "container-fluid"
+tagline: "25 July 2025, 佰仕廷酒店, 成都"
+layout: "single"
+tagline_suffix: |
+  <a class="btn btn-bordered padded fw-700 margin-top-40 margin-bottom-50" href="https://www.wjx.top/vm/mo396H3.aspx">演讲嘉宾征集</a>
```

Comment:
Same I would suggest here

```suggestion
  <div class="display-flex gap-10 margin-bottom-30">  
    <a class="btn btn-primary" href="https://www.wjx.top/vm/mo396H3.aspx">演讲嘉宾征集</a>  
  </div>  
```

Reply:
Thanks , that's OK to me, just like your version